src/status_im/ui/screens/wallet/transactions/subs.cljs

```diff
@@ -147,15 +148,15 @@
         (* 100 (/ confirmations max-confirmations))))))
 
 (reg-sub
-  :contacts-by-address
+    :contacts-by-address
   (fn [db]
     (into {} (map (fn [[_ {:keys [address] :as contact}]]
                     (when address
                       [address contact]))
                   (:contacts/contacts db)))))
 
 (reg-sub
-  :contact-by-address
+    :contact-by-address
```

Comment:
Weird alignement?

Reply:
doesn't matter will be removed by rebase I already fixed it in the other PR